radio/src/targets/flysky/CMakeLists.txt

```diff
@@ -51,10 +55,9 @@ set(EEPROM EEPROM_RLC)
 set(TARGET_DIR flysky)
 set(PPM_LIMITS_SYMETRICAL YES)
 add_definitions(-DPCBI6 -DBUZZER)
-add_definitions(-DPCBI6_USB_MSD)
+# add_definitions(-DPCBI6_USB_MSD)
```

Comment:
Shouldn't that be enabled by default?

Reply:
True... I disabled it for debugging and forgot it.